client/lib/analytics/ad-tracking.js

```diff
@@ -1155,6 +1157,55 @@ export function recordAliasInFloodlight() {
 	recordParamsInFloodlight( params );
 }
 
+/**
+ * Records a signup|purchase in Bing.
+ *
+ * @param {Object} cart - cart as `CartValue` object.
+ * @param {Number} orderId - the order ID.
+ * @returns {void}
+ */
+function recordOrderInBing( cart /*, orderId */ ) {
```

Comment:
Let's add a comment here to make sure others will understand that the orderId is passed, not yet used, but might be used in the future.

Reply:
Good suggestion. Done. Thanks!